### ISSUE TITLE
reports error

### ISSUE THREAD

**Issue by jrotieno** [broadinstitute/viral-ngs#496]
There seems to be an error in this stage of preparing the reports:

7 of 11 steps (64%) done
rule consolidate_fastqc_on_all_runs:
	output: reports/summary.fastqc.raw.txt
	wildcards: adjective=raw, dir=reports
usage: reports.py subcommand consolidate_fastqc [-h]
                                                inDirs [inDirs ...] outFile
reports.py subcommand consolidate_fastqc: error: the following arguments are required: outFile
Error in job consolidate_fastqc_on_all_runs while creating output file reports/summary.fastqc.raw.txt.

I noticed that in the reports.rules file;
rule consolidate_fastqc_on_all_runs:
    input:  
            expand("{{dir}}/fastqc/{sample}/{{adjective}}",
                sample=read_samples_file(config["samples_per_run"]))
    output: '{dir}/summary.fastqc.{adjective,raw|cleaned|taxfilt}.txt'

The output path may be wrong?

**Comment by tomkinsc** [comment]
Do you have sample names in`samples-runs.txt`? Can you tell if fastqc is being run? If not, `inDirs` may be empty (thus preventing report consolidation since there wouldn't be anything to consolidate). What do you have in `reports/fastqc` after running snakemake?

**Comment by jrotieno** [comment]
Hi Tom,
This time it ran perfectly. I think the problem may have had to do with not having sample names in sample-runs.txt.
I had omitted putting sample name therein as I thought it was the cause of errors I kept getting on instrahost/interhost analysis, even after commenting out the "includes" for these in the Snakemake file. It seems commenting out the line    #config["data_dir"]+'/'+config["subdirs"]["intrahost"] +'/isnvs.vcf.gz', is what may have solved the recurring intrahost/interhost error and not actually omitting sample name in sample-runs.txt. My bad!
